README.md

```diff
@@ -63,19 +171,19 @@ describe('module test', function () {
   beforeEach(function () {
     envStub = sinon.stub();
 
+    // Require our concat file, but stub out env-var
     mod = proxyquire('./concat', {
-      'env-var': envStub
+      'env-var': sinon.stub().returns({
```

Comment:
Instead of mocking up the entire chain, maybe it would be nice to have env-var provide a mock environment that can be used?

```js
var mod = proxyquire('./concat', {
  'env-var': require('env-var/test').mockEnv({VAR_NAME: 'world'})
});
```

It would require sinon to be in the main dependencies, though.